### ISSUE TITLE
Django-MPTT is unmaintained. Migrate to Django-Treebeard?

### ISSUE THREAD

**Issue by greyhare** [ellmetha/django-machina#306]
The [Django-MPTT](https://github.com/django-mptt/django-mptt) README says that it's currently maintained.

I'm looking at adding Machina to a website I'm designing, and nothing else uses MPTT. Everything tree-related (e.g. [Django CMS](https://www.django-cms.org/en/)) is using [Django Treebeard](https://github.com/django-treebeard/django-treebeard/).

Are there any plans to migrate?

**Comment by ellmetha** [comment]
Hey! 👋 I am not working actively on Django-machina at the moment, so I can't promise anything regarding this. I'll happily review any pull requests related to such migration though! 🙂

**Comment by greyhare** [comment]
Fair enough. Any general ideas on how to approach it? Like where to look for the parts that use MPTT now?

**Comment by ellmetha** [comment]
MPTT is used in order to implement the tree of forums (each forum can have child forums), which corresponds to the [`Forum`](https://github.com/ellmetha/django-machina/blob/main/machina/apps/forum/abstract_models.py#L31) model. This is where I would start looking in order to migrate to another equivalent library!
